Source/diablo.cpp

```diff
@@ -393,8 +393,7 @@ void RightMouseDown()
 		SetSpell();
 		return;
 	}
-	if (MousePosition.y <= RIGHT_PANEL_Y || MousePosition.y >= RIGHT_PANEL_Y + SPANEL_HEIGHT
-	    || ((!sbookflag || MousePosition.x <= RIGHT_PANEL_X || MousePosition.x >= RIGHT_PANEL_X + SPANEL_WIDTH)
+	if ((!sbookflag || !rectRightPanel.Contains(MousePosition))
 	        && !TryIconCurs()
 	        && (pcursinvitem == -1 || !UseInvItem(MyPlayerId, pcursinvitem)))) {

```

Comment:
This should fix the build
```suggestion
	        && (pcursinvitem == -1 || !UseInvItem(MyPlayerId, pcursinvitem))) {
```